rules/windows/defense_evasion_root_dir_ads_creation.toml

```diff
@@ -0,0 +1,51 @@
+[metadata]
+creation_date = "2024/03/14"
+integration = ["endpoint", "windows"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/03/14"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies the creation of an Alternate Data Stream (ADS) at the volume root directory (C:), which can indicate the
+attempt to hide tools and malware, as ADSs created in this directory are not displayed by system utilities.
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.sysmon_operational-*"]
```

Comment:
```suggestion
index = ["winlogbeat-*", "logs-endpoint.events.file-*", "logs-windows.sysmon_operational-*"]
```

Reply:
if you agree with suggestion to query both file and process events then maybe extend it to the system index

Reply:
Added both indexes.